### PR TITLE
Update geoipupdate to 5.1.1

### PR DESCRIPTION
**Pull request by williamdes**
I have read all changelogs between the two versions:
- https://github.com/maxmind/geoipupdate/releases
- https://github.com/maxmind/geoipupdate/releases?page=2

Since https://github.com/maxmind/geoipupdate/releases/tag/v4.8.0 there is two new ENVs to avoid overriding the entrypoint 
@chadwhitacre you did comment on #766 back in the day https://github.com/getsentry/self-hosted/pull/766/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3
Should we use ENVs now ?

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -243,7 +243,7 @@ services:
       timeout: 10s
       retries: 30
   geoipupdate:
-    image: "maxmindinc/geoipupdate:v4.7.1"
+    image: "maxmindinc/geoipupdate:v5.1.1"
```
Comment (chadwhitacre) [inline docker-compose.yml:246]: [Looks like](https://hub.docker.com/r/maxmindinc/geoipupdate) they are moving to [GHCR](https://github.com/maxmind/geoipupdate/pkgs/container/geoipupdate). Should we update here?

**Comment by chadwhitacre** [comment]
> Should we use ENVs now?

Seems reasonable. https://github.com/maxmind/geoipupdate/pull/143 does look like it addresses our use-case directly. :)

File: docker-compose.yml (hunk 1 above)
Comment (williamdes) [reply docker-compose.yml:246]: It will cause setup problems as the ghcr images require github ghcr docker login to pull

Comment (chadwhitacre) [reply docker-compose.yml:246]: Hrm, okay. Looks like 5.1.1 is on both so this is fine for now.

**Review by hubertdeng123 — APPROVED** [review]

**Comment by chadwhitacre** [comment]
Thanks for the contribution, @williamdes! :)

**Pull request merged**